packages/inference/test/global.d.ts

```diff
@@ -1,11 +1,13 @@
-interface CustomMatchers<R = unknown> {
-	closeTo(expected: number, precision: number): R;
-}
-
 declare global {
 	namespace Vi {
-		interface Assertion extends CustomMatchers {}
-		interface AsymmetricMatchersContaining extends CustomMatchers {}
+		interface Assertion {
+			// todo: fix the any
+			closeTo(expected: number, precision: number): any;
+		}
+		interface AsymmetricMatchersContaining {
+			// todo: fix the any
+			closeTo(expected: number, precision: number): any;
+		}
```

Comment:
to pass linting cc @pngwn

Reply:
exhibit a of why i hate linting :P

Reply:
This isn't the same btw but works well enough